test/puppetlabs/trapperkeeper/services/status/status_core_test.clj

```diff
@@ -2,10 +2,25 @@
   (:require [clojure.test :refer :all]
             [schema.core :as schema]
             [schema.test :as schema-test]
-            [puppetlabs.trapperkeeper.services.status.status-core :refer :all]))
+            [puppetlabs.trapperkeeper.services.status.status-core :refer :all]
+            [trptcolin.versioneer.core :as versioneer]))
 
 (use-fixtures :once schema-test/validate-schemas)
 
+(deftest get-service-version-test
+  (testing "get-service-version returns a version string that satisfies the schema"
+    ;; This test coverage isn't very thorough, but anything beyond this would
+    ;; really just be testing the underlying libraries that we use to
+    ;; implement it.
+    (is (nil? (schema/check
+                SemVerVersion
+                (get-service-version "puppetlabs" "trapperkeeper-status"))))
+    (is (thrown? IllegalStateException
```

Comment:
haha :) OK i'll improve the test to have it validate the exception more thoroughly.